packages/wasm/rust/Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
-name = "wasm"
+name = "lit-utilities-wasm"
```

Comment:
What's the difference between `wasm` and `lit-utilities-wasm`? and why did we change from one to another?

Reply:
I change the name for publishing to `cargo` since it would be a bit confusing if the package name just showed up as `wasm`.